ui/app/components/secret-list/totp-list-item.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class SecretListItemComponent extends Component {
```

Comment:
component names should match the file 
```suggestion
export default class TotpListItem extends Component {
```